README.md

```diff
@@ -53,236 +14,167 @@ After you're finished please remove all the comments and instructions!
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
+- [⭐ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+<!-- PROJECT DESCRIPTION -->
```

Comment:
_note: Since there is no open PR in the backend project for me to review, I'll leave my comments here:_

# Backend review

- [ ] Since you are already instructing your users to run the app with `rails server` this command is superfluous:

![image](https://user-images.githubusercontent.com/57421823/227376432-df82d67d-d680-4ea5-991f-b974007722e7.png)

Kindly remove it from your readme, in order to prevent confusion. 👍 

- [ ]  Nice routes and nice use of the `#resources` method. However, you are generating more routes than needed.

## Explanation

By using the `only:` keyword parameter, you can generate only the routes that you are using.

[[Look](https://launchschool.com/books/demystifying_rails/read/routes_and_resources)](https://launchschool.com/books/demystifying_rails/read/routes_and_resources):

![https://user-images.githubusercontent.com/57421823/201431229-f7861d5e-3462-4897-b6fa-1242581bf06f.png](https://user-images.githubusercontent.com/57421823/201431229-f7861d5e-3462-4897-b6fa-1242581bf06f.png)

This is an industry best practice.

## Example

For the motorcycles controller, I can see that you are only using index, create, show, update, and destroy.

So:

```ruby
resources :vehicles, only: [:index, :create, :show, :update, :destroy]

```

That way, your routes will be more consistent, and only the ones needed will be generated💪

- [ ] I can see you have a populated `seeds.rb` file.  Kindly, please include the command `rails db:seed` on your readme, at the end of this section:

![image](https://user-images.githubusercontent.com/57421823/227378260-f20d2a86-1009-415d-969c-eb0d36a91c15.png)

That way, after migrating the database, your users will have a populated one. 👍